extension/src/lib/chromeSettings.js

```diff
@@ -57,19 +70,119 @@ fluid.defaults("gpii.chrome.settings", {
                 flowManager: "ws://localhost:8081/browserChannel",
                 retryTime: 10
             }
+        },
+        contextMenuPanel: {
+            type: "gpii.chrome.contextMenuPanel",
+            options: {
+                model: "{settings}.model",
+                strings: {
+                    captions: "youtube captions",
+                    inputsLarger: "enhance inputs",
+                    rightClickToSelect: "right-click to select",
+                    selfVoicing: "text-to-speech",
+                    simplifiedUI: "reading mode",
+                    syllabification: "syllables",
+                    tableOfContents: "table of contents"
+                },
+                distributeOptions: {
+                    reset: {
```

Comment:
This is far too much material for a single options distribution. Please package it as a grade and distribute that instead